### DISCUSSION TITLE
A few questions regarding the extension...

### DISCUSSION THREAD

**Discussion by MrIDK22** [thunderclient/thunder-client-support#566] [Q&A]
Hi All,

Just got a few questions:

1. Is it possible for the right click to show the same menu as clicking the ellipsis (... icon)? Instead of having to aim the cursor at ... and click on it?
2. Is it possible to be able to have multiple tabs of "Run Collection" open? For example, have split editors let and right, and have editor on the right contain multiple tabs of Run Collection so that I can just change tab and Run All instead of having to go back to the Activity bar and search for the collection I'd like to run.
3. Is it possible to run the same collection / request multiple times?

Thanks!

**Comment by rangav** [comment]
Hi @MrIDK22 

answers below:

1. Currently not possible I could not find a way to integrate with right click, if you have any suggestions let me know.
2. Multiple tabs of Run Collection is possible, do you want to open multiple run collection views for different collections or same collection?
Or I can create a shortcut way to open run collection like `Ctrl + Click` will make it easy. 
3. Run collection multiple times - this feature is in roadmap.

**Reply by MrIDK22** [reply]
Hi @rangav,

1. I am not sure if this will help, but in Project Manager (another extension) they seem to be able to [add commands to the context menu](https://github.com/alefragnani/vscode-project-manager/blob/735f2caca141a70feac93ab22b75dc3fa4d9a06a/src/extension.ts#L102). The Activity Bar seems quite different compared to Thunder Client though.
2. Multiple tabs of different collections. `Ctrl + click` could work.

Thanks man!

**Reply by rangav** [reply]
Thanks for sharing, will look into it.
